scheduler/services/redis_client/__init__.py

```diff
@@ -1 +1,2 @@
 from .redis_client import redis_client
+from .redis_client import RedisClient
```

Comment:
The client is a singleton so calling the already instanced redis_client achieve the same.

Reply:
VSCode was complaining, it didn't find RedisClient, maybe just an editor mistake :cry:

Reply:
If is called like `redis_client.flatten_dict` instead of `RedisClient.flatten_dict` ? if not lets add it for peace of mind.